gitgud/__main__.py

```diff
@@ -264,12 +267,14 @@ def handle_load(self, args):
 
             if args.level_name is not None:
                 if args.level_name in all_skills[args.skill_name]:
+                    init_tracking_json(self.file_operator)
                     level = skill[args.level_name]
                     self.load_level(level)
                 else:
                     print('Level "{}" does not exist'.format(args.level_name))
                     print("To view levels/skills, use git gud levels or git gud skills")
             else:
+                init_tracking_json(self.file_operator)
```

Comment:
Same here, you can remove this once `init_tracking_json is in `load_level`

Reply:
Also a part of that section you deleted.